src/Sulu/Bundle/SecurityBundle/Tests/Functional/Controller/UserControllerTest.php

```diff
@@ -862,10 +862,13 @@ public function testGetUserAndRolesByContact(): void
         $this->assertEquals('admin', $response->username);
         $this->assertFalse(\property_exists($response, 'password'));
 
-        $this->assertEquals('Role1', $response->userRoles[0]->role->name);
-        $this->assertEquals('Sulu', $response->userRoles[0]->role->system);
-        $this->assertEquals('Role2', $response->userRoles[1]->role->name);
-        $this->assertEquals('Sulu', $response->userRoles[1]->role->system);
+        $names = [
+            $response->userRoles[0]->role->name,
+            $response->userRoles[1]->role->name,
+        ];
+
+        $this->assertContains('Role1', $names);
+        $this->assertContains('Role2', $names);
```

Comment:
This test is flacky so I refactored it as the order is not longer forced since latest changes:

https://github.com/sulu/sulu/actions/runs/9805891396/job/27076516928